core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -584,6 +584,8 @@
   REPLICATION_WORK_PROCESSOR_PERIOD("replication.work.processor.period", "0s", PropertyType.TIMEDURATION,
       "Amount of time to wait before re-checking for replication work, not useful outside of tests"),
   REPLICATION_TRACE_PERCENT("replication.trace.percent", "0.1", PropertyType.FRACTION, "The sampling percentage to use for replication traces"),
+  REPLICATION_RPC_TIMEOUT("replication.rpc.timeout", "2m", PropertyType.TIMEDURATION,
```

Comment:
If this is done for 1.7, then it's a behavior change in replication for a bug fix release.   I think its ok and good, we somehow need to remember to mention this in the 1.7 release notes.